tests/testthat/test-tab_style.R

```diff
@@ -473,4 +473,90 @@ test_that("using fonts in `cell_text()` works", {
     expect_match(
       "<td style=\"padding-top: 8px; padding-bottom: 8px; padding-left: 5px; padding-right: 5px; margin: 10px; border-top-style: solid; border-top-width: 1px; border-top-color: #D3D3D3; border-left-style: none; border-left-width: 1px; border-left-color: #D3D3D3; border-right-style: none; border-right-width: 1px; border-right-color: #D3D3D3; vertical-align: middle; overflow-x: hidden; text-align: left; font-family: 'Dancing Script', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;\">13:35</td>"
     )
+
+  gtcars_tbl <-
+    gtcars %>%
+    dplyr::filter(ctry_origin == "United Kingdom") %>%
+    dplyr::select(mfr, model, year, hp) %>%
+    gt()
+
+  # Expect no difference in output when using styles within a list or without
+  expect_equal(
+    gtcars_tbl %>%
+      tab_style(
+        style =
+          cell_text(
```

Comment:
This test needs a `fonts` argument with a non-single-element value

Reply:
Now added in.